app/packages/looker/src/worker/disk-overlay-decoder.ts

```
@@ -0,0 +1,137 @@
+    maskPathDecodingPromises.push(...promises);
+  }
+
+  const overlayFields = getOverlayFieldFromCls(cls);
+  const overlayPathField = overlayFields.disk;
+  const overlayField = overlayFields.canonical;
+
+  if (Boolean(label[overlayField]) || !Object.hasOwn(label, overlayPathField)) {
+    // it's possible we're just re-coloring, in which case re-init mask image and set bitmap to null
+    if (
+      label[overlayField] &&
+      label[overlayField].bitmap &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)